lib/std/os/linux.zig

```diff
@@ -1386,6 +1386,58 @@ pub fn madvise(address: [*]u8, len: usize, advice: u32) usize {
     return syscall3(.madvise, @ptrToInt(address), len, advice);
 }
 
+pub fn posix_fadvise(fd: fd_t, offset: off_t, len: off_t, advice: usize) usize {
```

Comment:
The naming is a bit unfortunate as the syscall is named `fadvise` while the libc wrapper is `posix_fadvise`...

Reply:
Ok, I'll rename to `fadvise`.